pages/apim/3.x/event-native/apim-event-native-api-management-activate-mode.adoc

```diff
@@ -0,0 +1,30 @@
+[[apim-event-native-api-management-activate-mode]]
+= Activating/deactivating the new execution engine mode
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_event_native_api_management_activate_mode.html
+:page-folder: apim/event-native
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+
+To activate the new event-native policy execution mode, you must enable it in the `gravitee.yml` file using the following new keys:
+
+* `JUPITER_MODE_ENABLED`. This option enables you to activate or deactivate the new event-driven execution engine mode on the Gateway. The key values can be `true` or `false`, respectively.
+* `JUPITER_MODE_DEFAULT`. This option let you define if the new mode will be the default one in which your new APIs will be created. The key values can be `never` or `always`. To make the new mode the default one, set the value to `always`. If you set to `never`, you will need to activate the new mode for each new API if you want to use it.
+
+You could switch this value individually per API by setting the `execution_mode` to "jupiter" or "v3" in your `API_Definition`, as shown in the example below.
```

Comment:
this is only true for api definition V2 and is not compliant with api definition V4.

Reply:
OK removing this line